interface/src/app/signature/participation-type-selector.tpl.html

```diff
@@ -0,0 +1,7 @@
+<div class="alert alert-danger" role="alert" ng-if="$ctrl.errorMessage">
+  {{$ctrl.errorMessage}}
+</div>
+<select name="selectedParticipationType"
```

Comment:
It might be good to put the description of the currently selected participation type immediately after the dropdown. Otherwise, the user is unlikely to understand what the different participation types mean.